portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,58 @@
+
+
+function makeQuiz(){
+    questions.forEach((currentQuestion,questionNumber)=>{
+        output.push(
+            `<input type="checkbox" id="question${questionNumber}" name="question${questionNumber}" 
+            value="yes">
+            <label for="question${questionNumber}">${currentQuestion}</label></br>`);
+    });
+    quizContainer.innerHTML=output.join('');
+}
+function showResults(){
+    let count = 0;
+    questions.forEach( (currentQuestion, questionNumber) => {
+        let currentValue = document.getElementById(`question${questionNumber}`);
+        if(currentValue.checked === true){
+            count++;
+        }
+    });
+    let resultsCode = `<h2>Your Results:</h2><p>Your score was ${count}</p>`;
+
+    if(count<= 1){
+        resultsCode += `<p>You are definitely not a cat. </p>`;
+    }else if (count <=7){
+        resultsCode += `<p>You are not a cat, but you do have some catlike tendencies.</p>`;
+    }else if (count <=10){
+        resultsCode += `<p>You might be a cat.</p>`;
+    }else{
+        resultsCode += `<p>You are a cat.</p>`;
+    }
+    resultsContainer.innerHTML = resultsCode;
+    window.scrollBy(000, 400);
+}
+
+const quizContainer = document.getElementById('quiz');
+console.log(quizContainer);
```

Comment:
It's a good idea to remove console.logs so the end user doesn't see them.

If you really need debugging statements to stay around, I would define a function:

```js
function LOG() {
    if (false) {
        console.log.apply(console, arguments);
    }
}
```

Use it like `LOG("List of words: ", words)`.

Then in development you can change the hardcoded false to true.

Reply:
Got it!